libr/bin/pdb/pdb.c

```diff
@@ -1105,7 +1307,7 @@ static void print_gvars(R_PDB *pdb, ut64 img_base, int format) {
 		return;
 	}
 	if (format == 'j') {
-		pdb->cb_printf ("{\"%s\":[", "gvars");
```

Comment:
Maybe port this to `pj` API too?

Reply:
Yea sure, the function seems much smaller than the print types so it won't be a problem

Reply:
Done